.github/workflows/changeset.yml

```diff
@@ -18,5 +18,6 @@ jobs:
         with:
           # Calls out to `changeset version`, but also runs prettier
           version: yarn release
+          title: Version Packages - plugins
```

Comment:
Want to add the `- plugins` into the PR title so the ticket that will be created will be named the same. It should make easier for us to decide which ticket is for which repo